javascript/javascript1/week2/StudentManager.js

```diff
@@ -0,0 +1,57 @@
+//Studen Manager
+
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+  // You write code here
+  // studentName ==' '|| studentName == null
+  if( typeof (studentName) !== "string" ){
```

Comment:
i belive you want something like this...Empty string `""`is still a valid string 
```suggestion
  if ( studentName !== "" ){
```

Reply:
ok Thanks 
done

Reply:
but after i did that changes if i add number it says ok and added it like student

Reply:
![Screenshot (5)](https://user-images.githubusercontent.com/75618135/106738167-e5c36180-6617-11eb-9c11-92db368b81aa.png)
![Screenshot (6)](https://user-images.githubusercontent.com/75618135/106738172-e6f48e80-6617-11eb-8ad2-a03f46fe4c67.png)
it is working fine with ( typeof (studentName) != "string"|| studentName ==' ') and i add or studentName ==' ' so it is avoid empty space